### PR TITLE
Merge master into next

### PR DESCRIPTION
**Pull request by na9da**
### What this PR does

Fixes #<insert issue number here if relevant>

Merges changes from `master` into `next` branch.

Following conflicts were resolved:
- CHANGES.md
- Translation files:
   - For new translation files from `master` that were not in `next` - I have added them to `next` after removing keys no longer in `next` or if their english translations don't match.
   - For existing translation files, I have imported translations from `master` if their english text match the one in `next`
- gulpfile.js - retained `next` version
- CesiumTimeline.jsx & TimeLine.jsx - Added the fix from master to resize timeline when resizing window.

### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [ ] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/Language/ja/translation.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
 {
   "tour": {
     "exploreMapDataButton": {
-      "content": "## データを地図上で見る\n\nここでは、データカタログからデータを探し、地図に追加することができます。複数のデータも追加できます。追加したデータは、画面左側にリストとして表示されます。"
+      "content": "## データを見る\n\nここでは、データカタログからデータを探し、地図に追加することができます。複数のデータも追加できます。追加したデータは、ワークベンチ内にリストとして表示されます。"
     },
     "locationSearchInput": {
-      "content": "## 場所を検索\n\n地名や住所を入力するとその場所を地図上で検索できます。"
+      "content": "## 場所検索\n\n地図上の興味ある場所の名前や住所を入力してください。"
```
Comment (tephenavies) [inline lib/Language/ja/translation.json]: How do you know which of these to keep?

File: lib/Language/fr/translation.json [hunk 2]
```diff
@@ -297,15 +294,9 @@
...
-    },
-    "mapboxVectorTile": {
-      "name": "Mosaïque vectorielle Mapbox"
-    },
-    "mapboxMap": {
-      "name": "Carte Mapbox"
-    },
+    "mapboxStyle": { "name": "Style Mapbox" },
+    "mapboxVectorTile": { "name": "Mosaïque vectorielle Mapbox" },
+    "mapboxMap": { "name": "Carte Mapbox" },
```
Comment (tephenavies) [inline lib/Language/fr/translation.json]: Can you reformat the translation files to match our other translation files? I think weblate is fine with stringified JSON with 2 spaces.

File: lib/Language/ja/translation.json (hunk 1 above)
Comment (na9da) [reply lib/Language/ja/translation.json]: Good catch!

The `master` branch don't have these translations, so these changes can't be from master. I suspect the changes are due to some encoding differences in my local machine. I have reverted the whole file to previous version.

File: lib/Language/fr/translation.json (hunk 2 above)
Comment (na9da) [reply lib/Language/fr/translation.json]: Prettier automatically did that, I have now reverted it to the original 2 spaces format.

**Review by tephenavies — APPROVED** [review]

**Comment by tephenavies** [comment]
CI deploy build failing because `TerriaMap@next` is out of date

**Pull request merged**